packages/swapper/src/types.ts

```diff
@@ -598,3 +604,7 @@ export type SupportedChainIds = {
 }
 
 export type MonadicSwapperAxiosService = ReturnType<typeof makeSwapperAxiosServiceMonadic>
+
+export enum MixPanelEvent {
+  RelayMultiSteps = 'Relay Multi Steps Quote',
```

Comment:
suggestion: multi hop vernacular and have the mixpanel reporting e.g something along the lines of `Unable to execute Relay multi-hop quote` for analytics sanity's sake

Reply:
Done in [7e3a115](https://github.com/shapeshift/web/pull/9188/commits/7e3a115c7aeb37843ca13844d6f4f73501a40f03)

Reply:
What about the second part re: reporting being something more human readable i.e 
> have the mixpanel reporting e.g something along the lines of Unable to execute Relay multi-hop quote for analytics sanity's sake

Reply:
Alright, agree, it's been changed in [29fc44d](https://github.com/shapeshift/web/pull/9188/commits/29fc44d8ce158086f2cbec2c3c7ee0dd44302dac)